en/SQLDatabase.md

```diff
@@ -12,8 +12,8 @@ If you used to work with an older version of JabRef, please refer to [SQLDatabas
 ## Usage
 
 To use this feature you have to connect to a remote database. 
-To do so you have to open **File** in the menu bar and then click the **Open shared database** item.
-The **Open shared database** dialog will open and you will have to fill in the shared's database connection settings.
+To do so you have to open **File** in the menu bar and then click the **Connect to shared database** item.
```

Comment:
@mlep  Best is to create a new issue in the JabRef repo as folllow up to 
JabRef/jabref#2009
becasuse the Screenshots would have to be changed, too